Remove legacy schema

wizardofoz123 opened the pull request:
This is an attempt to remove the old (legacy) db schema from kpet.

Commit:
drop schema conversion

Signed-off-by: Jakub Racek <jracek@redhat.com>

Commit:
rewrite assets db

Signed-off-by: Jakub Racek <jracek@redhat.com>

spbnick commented on kpet/schema.py:
```diff
@@ -158,43 +158,6 @@ def resolve(self, data):
-        # Resolve the data with the last schema
-        return last_valid_schema.resolve(data)
-
-
```
Please don't remove the Succession schema implementation, we'll keep using it :smile:!

spbnick commented on tests/assets/db/invalid/semantics/missing_case/suite.yaml:
```diff
@@ -1,6 +1,3 @@
     - name: case1
-patterns:
-    - pattern: a
-      case_name: case2
```
We need to remove this database and the associated test case altogether as they no longer apply. The test should have a TODO about this.

spbnick reviewed (changes requested):
Looks great, thanks a lot, Jakub! Requesting a couple tweaks only.

Commit:
remove invalid testcase

This testcase doesn't make sense anymore for generic matching.
Remove it, its assets and the TODO that says to remove it.

Signed-off-by: Jakub Racek <jracek@redhat.com>

spbnick reviewed (approved):
Looks good! Thank you, Jakub :)
